mne/io/bti/bti.py

```diff
@@ -914,8 +914,8 @@ def _read_bti_header(pdf_fname, config_fname, sort_by_ch_name=True):
                      in [c_['chan_no'] for c_ in chans]]
 
         # check all pdf channels are present in config
-        match = [c['chan_no'] for c in chans_cfg] == \
-                [c['chan_no'] for c in chans]
+        match = sorted([c['chan_no'] for c in chans_cfg]) == \
+                sorted([c['chan_no'] for c in chans])
```

Comment:
I suspect that this will not be enough, and that you actually need to reorder one to match the other. See a few lines below, we just `zip` over them -- this implies they are equivalently ordered already.

Reply:
Ah yes, I see that. That will be a problem. Do you think unmatched orders will be a problem elsewhere as well? I can work something to reorder these, although it probably won't be done as elegantly as the code deserves.

Reply:
Interestingly, it looks like there is a typo on line 718 as well. low_frew instead of low_freq. Ran into this problem after sorting when it got to line 1177 and is examining the process steps. Making a note here partially so that I don't forget to note it while I'm working on this.